protocol/migrations/sqlite/1679326850_add_community_token_owners.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS community_tokens_owners (
```

Comment:
nit: more grammatically correct :smile: 
```suggestion
CREATE TABLE IF NOT EXISTS community_token_owners (
```

Reply:
Fixed